_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
+| [Service-to-service mutual TLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Incremental Enablement of service-to-service mutual TLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
+| [VM: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [OPA Checker](https://github.com/mangchiandjjoe/istio/blob/d5390f6e436225949907d77ad3e9747a9bc26722/mixer/adapter/opa/README.md)  - **New to 0.4**      | Alpha 
+
 
-### User Interface
 
+### Core
 
-| Feature           | Phase       
+
+| Feature           | Phase        
 |-------------------|-------------------
-| [Grafana Dashboard](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html) | Alpha
-| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)    | Alpha
+| [Kubernetes: Istio Data Plane: Envoy Installation and Traffic Interception](https://istio.io/docs/setup/kubernetes/)        | Beta
+| [Kubernetes: Istio Control Plane Installation](https://istio.io/docs/setup/kubernetes/) | Beta
+| [Pilot Integration into Kubernetes Service Discovery](https://istio.io/docs/setup/kubernetes/)         | Beta
+| [Attribute Expression Language](https://istio.io/docs/reference/config/mixer/expression-language.html)        | Beta
+| [Mixer Adapter Authoring Model](https://istio.io/blog/posts/2017/adapter-model.html)        | Beta
+| [VM: Istio Data Plane: Envoy Installation, Traffic Interception and Service Registration](https://istio.io/docs/guides/integrating-vms.html)    | Alpha
+| [Kubernetes: Istio Control Plane Upgrade](https://istio.io/docs/setup/kubernetes/) | Alpha
+| [VM: Istio Control Plane: Installation and Upgrade (Galley, Mixer, Pilot, CA)](https://github.com/istio/istio/issues/2083)  | Alpha
+| [Pilot Integration into Consul, Eureka and Cloud Foundry Service Discovery](https://istio.io/docs/setup/consul/quick-start.html)     		   | Alpha
+| [Basic Config Resource Validation](https://github.com/istio/istio/issues/1894)         	   | Alpha
+| [Distributed Tracing to Zipkin / Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing.html)         | Alpha
 
 
-### Performance and Scalability
 
 
-| Feature           | Phase       
-|-------------------|-------------------
-| [Initial scenarios, manual baseline](https://docs.google.com/document/d/1LG5sZRwjL5l8h2e9dCbai27yr9gVezzsrJaHT6mICiM/edit#) | Alpha
-| [Reproducible oss characterization tool (Fortio)](https://github.com/istio/fortio/issues)    | Alpha
-| [Mixer bottom up perf](https://github.com/istio/old_mixer_repo/issues/1472)    | Alpha
 
-Please get in touch by joining our [community]({{home}}/community) if there are features you'd like to see in our future releases!
+**Please get in touch by joining our [community]({{home}}/community) if there are features you'd like to see in our future releases!**
```

Comment:
Please remove the **, this doesn't need to be bolded.

Reply:
I changed it to bold after a conversation with Varun who was asking me how people can request new features - he was looking at the page but didn't notice the text. I was hoping changing it to bold will make it stand out.

Reply:
I changed it to bold after a conversation with Varun who was asking me how people can request new features - he was looking at the page but didn't notice the text. I was hoping changing it to bold will make it stand out.